packages/console/src/pages/TenantSettings/Subscription/PlanComparisonTable/renderers/GeneralFeatureFlag.tsx

```diff
@@ -0,0 +1,20 @@
+import { type Optional } from '@silverhand/essentials';
+
+import Success from '@/assets/icons/success.svg';
+import DynamicT from '@/ds-components/DynamicT';
+
+import QuotaValueWrapper from './QuotaValueWrapper';
+
+type Props = {
+  isEnabled: Optional<boolean>;
+};
+
+function GeneralFeatureFlag({ isEnabled }: Props) {
```

Comment:
What is `general feature`? Maybe `generic`?

Reply:
Yes, use `generic` instead.